sources/packages/backend/apps/workers/src/controllers/supporting-user/_test_/e2e/supporting-user.controller.createSupportingUsers.e2e-spec.ts

```diff
@@ -154,4 +226,24 @@ describe("SupportingUserController(e2e)-createSupportingUsers", () => {
       SupportingUserType.Partner,
     );
   });
+
+  /**
+   * Get a notification record for supporting user information notification message.
+   * @returns notification record.
+   */
+  async function getCreatedNotification(): Promise<Notification> {
+    const notifications = await db.notification.find({
+      select: {
+        id: true,
+        messagePayload: true,
+      },
+      where: {
+        notificationMessage: {
+          id: 29,
+        },
+      },
+      order: { id: "DESC" },
+    });
+    return notifications[0];
+  }
```

Comment:
I believe when the userId of the student is put back into notification, then it is more accurate to get notification by notificationmessage id and user id. Also findOne is more suitable to get single result.